design-proposals/emts-scale-provisioning.md

```diff
@@ -0,0 +1,321 @@
+
+Once the process is taken over by iPXE (Step 10), the `boot.ipxe` is downloaded by iPXE from the local HTTP server and chain-loaded (**Steps 11-15**).
+The `boot.ipxe` script downloads Micro-OS from the local HTTP server and boots into it (**Steps 16-17**).
+The Micro-OS has all necessary certificates to communicate with the central orchestrator. Micro-OS services start device discovery and Tinkerbell workflow to provision target OS.
+From this point, EN goes through standard provisioning process.
+
+### EIM standalone
+
+The provisioning of EMT-S at scale will be driven by a local orchestrator instance that only includes EIM components. 
+The main requirement of EIM standalone is to make it easy and resource-efficient for customers to deploy.
+
+There are two approaches (see options below) to how we can configure EIM standalone to be more lightweight and faster to deploy.
+
+This proposal makes a decision to proceed with Option 1 for 3.1 release, but Option 2 should be a target solution for EIM-S.
+
+#### Option 1: A slimmed down deployment based on the current EIM Standalone profile
+
+This approach will use the existing EIM standalone profile. This is an EMF deployment profile that only deploys EIM with Observability and FPS components.
+
+#### Option 2: A minimal EIM deployment
+
+Even the slimmed down version of EIM from Option 1 may not meet requirements for fast and easy deployment on-site.
+
+This option is a radical approach to make EIM super-lightweight and reduce any FPS dependencies that are not essential to EIM. 
+Note that this option requires some modifications to EIM components, as some of them rely on FPS components (e.g., Vault or Keylocak).
+
+The minimal EIM deployment is mainly used to support OXM use cases, where EIM is only used to provision ENs that are further shipped to the target location.
+
+The minimal EIM deployment at high level:
+- No cluster and application orchestrator deployed.
+- Simplified deployment with just Helm on a standard K8s cluster. No ArgoCD, Fleet, RKE2.
+- A single tenant deployment, without the Multi-Tenancy/IAM framework. A default tenant will be created to "own" EIM resources.
+- Reduced EIM with infra-core (without exporter, Tenant Controller), infra-onboarding and infra-managers
+- Minimal set of FPS services. A rough minimum of FPS services that may be required by EIM are:
+  - Postgres database
+  - RS proxy
+  - Vault (?)
+  - Traefik
+  - cert-manager
+  - MetalLB
+- Relaxed security requirements:
+  - No Istio for inter-Pod communication security
+  - No Kyverno for policies
+- No UI deployed, the EN pre-registration is done via Bulk Import Tool, API or CLI tool.
+- Limited observability stack that should only provide logs. No alerting, SRE, Prometheus.
+- Abandon HA requirements - EIM-S is primarily a single-node, on-prem deployment. Any data loss can be backed by hardware-level redundancy. Think of EIM-S as Rancher Desktop or Virtualbox.
```

Comment:
we dont have any HA mechanism when deployed locally

Reply:
Even in the observability stack ?